src/Team-Services-Bot.Api/Controllers/AuthorizeController.cs

```diff
@@ -91,14 +92,14 @@ public class AuthorizeController : Controller
                 data.SetCurrentProfile(result);
                 data.SetProfiles(profiles);
                 await this.botService.SetUserData(channelId, userId, data);
+
+                return this.View(new Authorize(pin));
             }
             catch (Exception ex)
```

Comment:
AFAIK there is no need to catch exception here and track it as it should be done automatically by AI request processing handler injected into MVC. It tracks exceptions in correlation with request processing,